website/content/docs/troubleshoot/common-errors.mdx

```diff
@@ -147,6 +149,40 @@ You have installed an Enterprise version of Consul. If you are an Enterprise cus
 
 -> **Note:** Enterprise binaries can be identified on our [download site][releases] by the `+ent` suffix.
 
+## Common errors on Kubernetes
+
+### Unable to connect to the Consul client on the same host
+
+```text
+Put http://10.20.62.49:8500/v1/catalog/register: dial tcp 10.20.62.49:8500: connect: connection refused
+```
+
+```text
+Put http://172.16.30.51:8500/v1/agent/service/register: dial tcp 172.16.30.51:8500: connect: connection refused
+```
+
+```text
+Get http://10.137.120.213:8500/v1/status/leader: dial tcp 10.137.120.213:8500: i/o timeout
```

Comment:
Does it makes sense to generalize these IP values or mention something along the lines of "where 10.20.62.49" is the cluster IP of the client?

Reply:
Hmm yeah, I think probably both are good. So maybe we use an IP like `10.0.0.10` or something and also clarify below what that IP is. wdyt?

Reply:
LGTM 😄

Reply:
Approving it assuming those changed will make it and not block this PR 👍